scrapy/__init__.py

```diff
@@ -7,7 +7,7 @@
 
 # Scrapy version
 import pkgutil
-__version__ = pkgutil.get_data(__package__, 'VERSION').decode('ascii').strip()
+__version__ = (pkgutil.get_data(__package__, 'VERSION') or b'').decode('ascii').strip()
```

Comment:
why is this change needed? Is it related to typing?

Reply:
Indeed:
```
scrapy/__init__.py:10: error: Item "None" of "Optional[bytes]" has no attribute "decode"
```

I just reverted it and added the file to the ignore list